### PR TITLE
Solaris10

### PR DESCRIPTION
**Pull request by dariaphoebe**
deal with older versions of tr and lex that exist on solaris 10.

### CODE REVIEW

**Comment by jaltman** [comment]
@lhoward please review

File: lib/libedit/src/makelist [hunk 1]
```diff
@@ -127,7 +127,7 @@ case $FLAG in
 #
 -fh)
     cat $FILES | $AWK '/el_action_t/ { print $3 }' | \
-    sort | tr '[:lower:]' '[:upper:]' | $AWK '
+    sort | tr [a-z] [A-Z] | tr -d ' ' | $AWK '
```
Comment (nicowilliams) [inline lib/libedit/src/makelist]: Missing quotes.

File: lib/com_err/lex.l [hunk 2]
```diff
@@ -53,8 +53,6 @@ static int getstring(void);
 
 %}
 
-%option nounput
```
Comment (nicowilliams) [inline lib/com_err/lex.l]: Does this cause problems on systems where lex generates an `unput()`?

File: lib/libedit/src/makelist (hunk 1 above)
Comment (nicowilliams) [reply lib/libedit/src/makelist]: That is, the `[a-z]` and `[A-Z]` need to be `'[a-z]'` and `'[A-Z]'`.

File: lib/com_err/lex.l (hunk 2 above)
Comment (lhoward) [reply lib/com_err/lex.l]: Probably will just cause a warning?

Comment (lhoward) [reply lib/com_err/lex.l]: Could be also useful to check for flex in configure, and if we're using it, pass --nounput on the command line. Do we want Heimdal to be able to compile without unused function warnings?

**Comment by dariaphoebe** [comment]
Managed to submit as a new PR, sorry.

#454

**Comment by nicowilliams** [comment]
Pushed via #454.

**Pull request closed without merging**